css-animations-2/Overview.bs

```diff
@@ -170,7 +170,7 @@ is <code>elem</code>. <code>animation</code> is disassociated from
 <pre class="example lang-javascript">
 elem.style.animation = 'spin 1s';
 let animation = elem.getAnimations()[0]; // animation's owning element is elem
-elem.style.animation = ''; // animation no longer has an owning element
+elem.style.animation = ''; // animation no longer has an owning element <!-- '' (Bikeshed syntax highlighting fix)-->
```

Comment:
Bikeshed syntax highlighting fix? Is this a bikeshed cheat code or a reminder?

Reply:
Oh, I should remove that! My syntax highlighter was treating the whole file as being part of a string starting on that line...